Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 autoexamples = false
 
 # also update in README.md (badge and "Rust version requirements" section)
-rust-version = "1.48"
+rust-version = "1.50"
```

Comment:
The MSRV bump shouldn't be neccesary once #1549 is merged.

Reply:
Well, you can see there is an error now: https://github.com/Geal/nom/actions/runs/3040066364

Reply:
This PR fix error and also add some improvements.

Reply:
Yes, and #1549 fixes that by using `min()` instead of `clamp()`.

Reply:
Ahh, sorry, I see that!

Reply:
Well, I don't see a problem to drop support for rust version which had a release 2 years ago.

Reply:
@ikrivosheev It is a huge problem, because it would require a major version bump. Quoting the README:

> The 7.0 series of nom supports **Rustc version 1.48 or greater**. It is known to work properly on Rust 1.41.1 but there is no guarantee it will stay the case through this major release.
>
> The current policy is that this will only be updated in the next major nom release.

MSRV guarantees are serious business for widely used libraries. It's fine if your library doesn't commit to a MSRV, but if your library does make such a commitment, you cannot break it.

Reply:
Hopefully 8.0 changes the MSRV guarantee to minor compatibility breaks (8.x.0) rather than major (x.0).  This is the approach of most libraries I work with and it seems to work out well.  Having more MSRV smarts in cargo would he,lp though.

Reply:
MSRV will probably change for nom 8 yes, but will stay the same for nom 7. Contrary to some libraries that bump MSRV in minor versions, we can't do that for nom because those versions have to be shipped in software for distributions like debian